### DISCUSSION TITLE
[closed][manuscript][fairtracks] Change from EFO to UBERON (and others)?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#123] [Team Posts]
@dzerbino @jmfernandez It seems over 10000 UBERON terms have been purged from EFO in the most recent version (https://raw.githubusercontent.com/EBISPOT/efo/master/ExFactor%20Ontology%20release%20notes.txt). Which means that we should probably use UBERON directly for organism_part instead of EFO. 

Also, I just argued (https://github.com/fairtracks/fairtracks_standard/issues/22) that “Most ontologies should be backwards compatible”, in a discussion about ontology versioning. Suddenly doing such massive changes to the EFO ontology, is a counter-argument, meaning that either we assume massive changes might happen anytime, or we should perhaps should not use EFO at all, only the raw ontologies it imports. I would not assume those others are so volatile, and am then currently in favor of ditching EFO completely.

**Comment by jmfernandez** [comment]
From my point of view, imported terms from other ontologies are not so safe as "native" ones, because the authors from the ontology can decide not to rely on them in the future.

**Comment by sveinugu** [comment]
In https://github.com/fairtracks/fairtracks_standard/pull/35, EFO has been changed in into UBERON for organism_part, but kept for cell lines and cell types as it seems to me that EFO is good for those topics. Closing this

**Comment by sveinugu** [comment]
Added link in https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/41.
